### ISSUE TITLE
Get Collections - Token Issue[Data]

### ISSUE THREAD

**Issue by DaniCana** [iexg/IEX-API#1431]
### Please keep all posted issues professional and constructive. We may delete posts that do not provide sufficient information, are outright negative and unproductive, or are not directly related to IEX Cloud.

Hello - I have just started to use IEX Cloud with python.
**Clearly describe the data problem**
I am trying to pull all the stocks from a list of sectors using "get_collections(sector)" where sector is the list of sectors I have defined. 
Unfortunately, I receive this error "The query could not be completed. Invalid auth token". 

This is very strange as my token works fine when pulling other data (i.e. stock quotes, list of all the symbols, etc.).

Can you please help me with this problem?

Thank you.
Dan
**Data URL (The URL you used to access the data without your token):**

**What time did the problem occur (including timezone)?**
It happened on the 28th of March at 11am.

**What is the expected behavior?**
Basically, I was trying to replicate what it is written in this page "https://addisonlynch.github.io/iexfinance/stable/stocks.html#stocks-collections"

**What are the steps to reproduce**

**Your environment (e.g. browser, OS, API version, language):**
I am using python on windows 10.

**Additional context**

**Comment by addisonlynch** [comment]
Hello, developer of ``iexfinance`` here. ``iexfinance`` is a 3rd-party library which is not officially supported by IEX Cloud. Please open an issue at https://github.com/addisonlynch/iexfinance if your problem persists. Thanks!
